src/auth.coffee

```diff
@@ -36,4 +36,4 @@ module.exports =
   #
   # token - A string token to save.
   saveToken: (token) ->
-    keytar.setPassword(tokenName, 'atom.io', token)
+    keytar.setPassword(tokenName, 'pulsar-edit.dev', token)
```

Comment:
⚠️ Ditto, this will also probably cause users to have to re-auth to the Pulsar API, or find and copy their existing token. Likely to cause some support requests. But I again lean toward thinking this is the right thing to do. Not clobber (over-write) the existing Atom token users might have, and make our token findable searching for "Pulsar" instead of "Atom".

Reply:
Again, happy to bow to wisdom here so if it is ok to make the change then lets do it.

Reply:
Okay, the poll was up for a week on Discord, we got no-one voting to re-use old the Atom token, six people voting to use a new token, even if it meant they had to re-log-in once.

I already felt this was the right thing to do, but it eases the decision to get the poll results IMO.